Cargo.toml

```diff
@@ -69,11 +68,13 @@ vm-fdt = "0.3"
 tempfile = "3.19.1"
 uuid = { version = "1.16.0", features = ["fast-rng", "v4"]}
 clean-path = "0.2.1"
+thiserror = "2.0.9"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 kvm-bindings = "0.11"
 kvm-ioctls = "0.21"
 vmm-sys-util = "0.12"
+landlock = "0.4.1"
 
```

Comment:
Nitpick: It's not fully alphabetically sorted at the moment, but it'd be nice if you could try to place the lines as alphabetically as possible :wink: